AntennaTracker/wscript

```diff
@@ -8,6 +8,7 @@ def build(bld):
         vehicle=vehicle,
         libraries=bld.ap_common_vehicle_libraries() + [
             'PID',
+            'AP_Avoidance',
```

Comment:
So there's a call in https://github.com/ArduPilot/ardupilot/pull/4278/commits/12152e864ca07dd9da054af135fcac9adb9a6310 to register obstacles from MAVLink packets (GPS_GLOBAL_INT packets in particular).  We do this there because the vehicle code ordinarily only sees packets to itself.  and (b) I didn't want to perpetuate/use the "snoop" function.  

I will change this so anyone who implements "handleMessage" must also implement "enjoyForwardedMessage(&msg)", which they can use to update themselves based on a packet just being forward.  This will remove the need to link AP_Avoidance in to all vehicles.  My only concern is the naming.